packages/meta-harness/tests/test_redaction.py

```
@@ -54,6 +54,20 @@ def test_openai_api_key_assignment_is_masked(self) -> None:
 
+    def test_broker_run_token_is_masked(self) -> None:
+        """Issue #354: broker の per-run トークン（mh- + token_urlsafe(24) = 32 文字）が
+        judge 失敗時の stdout 抜粋経由で artifacts へ永続化されうるため、mh- prefix の
+        トークン形式をマスクする。"""
+        token = "mh-" + "Ab3dEf6hIj9lMn2pQr5tUv8xYz1B4c6d"
+        result = redaction.redact_secrets(f"ANTHROPIC_API_KEY={token} rejected")
+        assert token not in result
+        assert "[REDACTED:meta-harness broker run token (mh-)]" in result
+
+    def test_short_mh_prefixed_names_are_not_masked(self) -> None:
+        """mh-run- 等の短いコンテナ名・エイリアスまで塗りつぶさないこと（20 文字未満）。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)